experiments/superdarn_common_fields.py

```diff
@@ -87,21 +87,21 @@
 
 # set sounding frequencies
 if opts.site_id == "sas":
-    SOUNDING_FREQS = [10300, 11000, 11700, 13250]
-    #SOUNDING_FREQS = [9500, 10300, 11000, 11700, 13250, 14200, 15200]
+    #SOUNDING_FREQS = [10300, 11000, 11700, 13250]
+    SOUNDING_FREQS = [9500, 10300, 11000, 11700, 13250, 14200, 15200]
 elif opts.site_id == "pgr":
-    SOUNDING_FREQS = [10350, 11050, 11750, 13300]
-    #SOUNDING_FREQS = [9500, 10300, 11000, 11700, 13250, 14200, 15200]
+    #SOUNDING_FREQS = [10350, 11050, 11750, 13300]
+    SOUNDING_FREQS = [9550, 10350, 11050, 11750, 13300, 14250, 15250]
 elif opts.site_id == "rkn":
-    SOUNDING_FREQS = [10400, 11100, 11800, 13450]
-    #SOUNDING_FREQS = [9500, 10300, 11000, 11700, 13250, 14200, 15200]
+    #SOUNDING_FREQS = [10400, 11100, 11800, 13450]
+    SOUNDING_FREQS = [9600, 10400, 11100, 11800, 13450, 14300, 15300]
 elif opts.site_id == "inv":
-    SOUNDING_FREQS = [10450, 11150, 11850, 13500]
-    #SOUNDING_FREQS = [9500, 10300, 11000, 11700, 13250, 14200, 15200]
+    #SOUNDING_FREQS = [10450, 11150, 11850, 13500]
+    SOUNDING_FREQS = [9650, 10450, 11150, 11850, 13500, 14350, 15350]
 elif opts.site_id == "cly":
-    SOUNDING_FREQS = [10550, 11200, 11900, 13550]
-    #SOUNDING_FREQS = [9500, 10300, 11100, 11700, 13250, 14200, 15200]
+    #SOUNDING_FREQS = [10550, 11200, 11900, 13550]
+    SOUNDING_FREQS = [9700, 10550, 11200, 11900, 13550, 14450, 15400]
```

Comment:
```suggestion
    SOUNDING_FREQS = [9700, 10500, 11200, 11900, 13550, 14400, 15400]
```

I think these are not restricted in the new list